documentation/src/main/asciidoc/chapter-4-retrieving-a-mapper.asciidoc

```diff
@@ -102,7 +102,7 @@ A mapper which uses other mapper classes (see <<invoking-other-mappers>>) will o
 [[injection-strategy]]
 === Injection strategy
 
-When using <<using-dependency-injection,dependency injection>>, you can choose between field and constructor injection.
+When using <<using-dependency-injection,dependency injection>>, you can choose between constructor, field, or method injection (the latter is only supported when using Spring).
```

Comment:
Why are you saying that method inject is only supported with Spring?

Reply:
My first implementation was only targeting Spring and there is only a test for Spring. I guess with your changes this might work for other frameworks as well. In that case a test for those should be added as well.

BTW: I called it method injection though I guess that is wrong and should be called setter injection?

Reply:
Indeed setter injection is better, thanks for changing it.

Yes my changes made it work in general. Will push some jsr330 and jakarta tests as well